src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -811,6 +811,13 @@ XBUtilities::string_to_bytes(std::string str)
   return size;
 }
 
+bool 
+XBUtilities::
+isPowerOf2(const uint64_t x)
+{
+  return !(x==0 || (x & (x - 1)));
```

Comment:
Candidate for being inline as it does nothing really.  Also either stick with equality check for both numbers or none at all for both, either is fine.

Reply:
I added the equality check for both. Are we comfortable with placing the definition of the function into the Utilities header file?